src/helloSwap.ts

```diff
@@ -97,24 +101,36 @@ export class HelloSwap {
             this.getNewSwaps().then(
                 (swaps: EmbeddedRepresentationSubEntity[]) => {
                     if (swaps.length) {
-                        console.log(
-                            `[${whoAmI}] ${swaps.length} new swap(s) waiting for a decision`
+                        this.logger[whoAmI](
+                            `${swaps.length} new swap(s) waiting to be started`
                         );
                     }
                     swaps.forEach(
                         async (swap: EmbeddedRepresentationSubEntity) => {
                             const simpleSwap = HelloSwap.toSwap(swap);
                             if (this.acceptPredicate(simpleSwap)) {
                                 await this.acceptSwap(simpleSwap);
-                                console.log(
-                                    `[${whoAmI}] swap accepted:`,
-                                    simpleSwap.id
+                                this.logger[whoAmI](
+                                    `Started swap with ID: ${simpleSwap.id}`
+                                );
+                                this.logger.data(
+                                    `with parameters: sell ${JSON.stringify(
+                                        simpleSwap.sellAsset
+                                    )}, buy ${JSON.stringify(
+                                        simpleSwap.buyAsset
+                                    )}`
                                 );
                             } else {
                                 await this.declineSwap(simpleSwap);
-                                console.log(
-                                    `[${whoAmI}] swap declined:`,
-                                    simpleSwap.id
+                                this.logger[whoAmI](
+                                    `Cancelled swap with ID: ${simpleSwap.id}`
+                                );
+                                this.logger.data(
+                                    `with parameters: sell ${JSON.stringify(
```

Comment:
I think in Javascript you can implement a `display` function that is used when doing display things to avoid doing `JSON.stringify` inline. It may be interesting to look into it.

Reply:
Briefly looked into this, but couldn't find anything. Would be better though!

Reply:
We could also use format strings: "sell %s", that is very common with loggers.

Reply:
Oh yeah, I think I know how to do this now with `winston`.

Reply:
`JSON.stringify` is no more!